colorbleed/plugins/fusion/inventory/select_loader_color.py

```diff
@@ -0,0 +1,40 @@
+from avalon import api
+from avalon.vendor.Qt import QtGui, QtWidgets
+
+import avalon.fusion
+
+
+class FusionSelectLoaderColor(api.InventoryAction):
+    """Update the color of the selected tools"""
+
+    label = "Select Loader Color"
```

Comment:
Let's refactor this to **Set Tool Color** - also use "FusionSetToolColor" for the top action, because if additional containers are to be supported it should end up working for those as well.